schema/current/MNN_generated.h

```diff
@@ -247,11 +247,12 @@ enum OpType {
   OpType_If = 601,
   OpType_LayerNorm = 603,
   OpType_GridSample = 604,
+  OpType_ROIAlign = 605,
```

Comment:
New Op type need be added before OpType_Plugin, Better use.OpType_ROIAlign = 143

Reply:
get it